### PR TITLE
Add new Uma model, texture and animations

### LINKED ISSUE #1212: [REFACTOR] Add new Uma Model and Animations
## 🚧 Refactor

Change the Old Uma model and Animations to the new ones.

### PR DESCRIPTION
**Pull request by BertovDev**
Closes #1212 

## Motivation

Add new Uma model, textures and animations.

## Summary of changes

1. Add new Uma Model, textures, Animations
2. Modify Model position
3. Create new Uma prefab variant

## How has this been tested?

To test it add "Uma" to the list `enabledCharactersName` of string inside the `MainScreenManager`. You will only be avaible to visualize her in the MainScreen model and characterInfo since she it is not implemented in the backend.

## Test additions / changes

List tests added/updated.

## Checklist
- [x] I have tested the changes locally.
- [x] I self-reviewed the changes on GitHub, line by line.
- [ ] Tests have been added/updated.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.
- [ ] I have tested the changes in another devices.
  - [ ] Tested in iOS.
  - [ ] Tested in Android.

### CODE REVIEW

**Comment by tkz00** [comment]
A conflict was generated. Also, this PR only affects the main screen model for Uma? Or we should also test it in game?

**Comment by BertovDev** [comment]
> A conflict was generated. Also, this PR only affects the main screen model for Uma? Or we should also test it in game?

Affects all the Uma models, UI and InGame model.

File: client/Assets/Prefabs/Characters/UICharacters/Muflus.prefab
Comment (AngieDutra) [inline client/Assets/Prefabs/Characters/UICharacters/Muflus.prefab:1]: Why were changes made to muflus?

Comment (BertovDev) [reply client/Assets/Prefabs/Characters/UICharacters/Muflus.prefab:1]: I rescale him having the references of the new Uma. We saw this with @saugon and we thought this change could fit better.

**Review by AngieDutra — APPROVED** [review]
LGTM!

**Pull request merged**